src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -35,71 +35,73 @@
-					<template #icon>
-						<FilterIcon :size="15" />
-					</template>
-					<NcActionButton close-after-click
-						class="filter-actions__button"
-						:class="{'filter-actions__button--active': isFiltered === 'mentions'}"
-						@click="handleFilter('mentions')">
-						<template #icon>
-							<AtIcon :size="20" />
-						</template>
-						{{ t('spreed','Filter unread mentions') }}
-					</NcActionButton>
-
-					<NcActionButton close-after-click
-						class="filter-actions__button"
-						:class="{'filter-actions__button--active': isFiltered === 'unread'}"
-						@click="handleFilter('unread')">
-						<template #icon>
-							<MessageBadge :size="20" />
-						</template>
-						{{ t('spreed','Filter unread messages') }}
-					</NcActionButton>
-
-					<NcActionButton v-if="isFiltered"
-						close-after-click
-						class="filter-actions__clearbutton"
-						@click="handleFilter(null)">
-						<template #icon>
-							<FilterRemoveIcon :size="20" />
-						</template>
-						{{ t('spreed', 'Clear filters') }}
-					</NcActionButton>
-				</NcActions>
-			</div>
-
-			<!-- Actions -->
-			<div class="actions"
-				:class="{'hidden-visually': isFocused}">
-				<NcActions class="conversations-actions">
-					<template #icon>
-						<DotsVertical :size="20" />
-					</template>
-					<NcActionButton v-if="canStartConversations"
-						close-after-click
-						@click="showModalNewConversation">
+			<transition-group name="radial-reveal">
+				<!-- Filters -->
+				<div v-show="!isFocused" key="filters" class="filters">
+					<NcActions class="filter-actions"
+						:primary="isFiltered !== null">
 						<template #icon>
-							<Plus :size="20" />
+							<FilterIcon :size="15" />
 						</template>
-						{{ t('spreed','Create a new conversation') }}
-					</NcActionButton>
+						<NcActionButton close-after-click
+							class="filter-actions__button"
+							:class="{'filter-actions__button--active': isFiltered === 'mentions'}"
+							@click="handleFilter('mentions')">
+							<template #icon>
+								<AtIcon :size="20" />
+							</template>
+							{{ t('spreed','Filter unread mentions') }}
+						</NcActionButton>
+
+						<NcActionButton close-after-click
+							class="filter-actions__button"
+							:class="{'filter-actions__button--active': isFiltered === 'unread'}"
+							@click="handleFilter('unread')">
+							<template #icon>
+								<MessageBadge :size="20" />
+							</template>
+							{{ t('spreed','Filter unread messages') }}
+						</NcActionButton>
+
+						<NcActionButton v-if="isFiltered"
+							close-after-click
+							class="filter-actions__clearbutton"
+							@click="handleFilter(null)">
+							<template #icon>
+								<FilterRemoveIcon :size="20" />
+							</template>
+							{{ t('spreed', 'Clear filters') }}
+						</NcActionButton>
+					</NcActions>
+				</div>
 
-					<NcActionButton close-after-click
-						@click="showModalListConversations">
+				<!-- Actions -->
+				<div v-show="!isFocused"
```

Comment:
Maybe put them into one `<div>` wrapper with pos: absolute and display: flex, as they are following the same logic?
then `<transition-group>` should be replaces with `<transition>`, and i think, it should work without `key` param

Reply:
that leads to a different animation as it will be counted on the wrapper and not on each button but let's wait for @marcoambrosini to check whether to keep it or not.

https://github.com/nextcloud/spreed/assets/84044328/574243bc-ee0f-4749-bd49-e8ee2d0a4a78

Reply:
Looks good, as discussed we've increased the animation speed